README.md

```diff
@@ -52,3 +56,27 @@ issue will be addressed in a future version.
 You can become root with `$ sudo ...` without needing a password. If you ever
 really need passwords for some accounts, note that the default Vagrant settings
 are still working, i.e. `root:vagrant` and `vagrant:vagrant`.
+
+## Mail server
+
+You do not need to change anything if you like to send emails via the
+self-administration. But if you like to read the sent mails, you need to use
+telnet. This is useful if you like to register a user and want to activate him
+with the generated link which is sent with the e-mail.
+
+Login to the greenmail server via telnet
+
+    $ telnet localhost 10110
+    
+Use the e-mail address which the self-administration is using for notification
+
+    $ USER e-mail address e.g. hello@osiam.org
```

Comment:
this is not a shell command, so no "$" needed before the "command". Does telnet print a prompt character? Then you should use it.